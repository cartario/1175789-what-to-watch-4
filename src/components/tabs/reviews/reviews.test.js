import React from "react";
import renderer from "react-test-renderer";
import Reviews from "./reviews.jsx";
import {Provider} from "react-redux";
import configureStore from "redux-mock-store";

const mockStore = configureStore([]);

describe(`Reviews-test`, () => {
  const store = mockStore({
    FILMS: {
      comments: [],
    },
  });
  it(`should render reviews`, () => {
    const tree = renderer.create(
        <Provider store = {store}>
          <Reviews

          />
        </Provider>
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
