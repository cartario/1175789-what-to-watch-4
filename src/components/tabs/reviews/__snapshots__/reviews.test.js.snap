// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews-test should render reviews 1`] = `
<div
  className="movie-card__reviews movie-card__row"
>
  <div
    className="movie-card__reviews-col"
  />
  <div
    className="movie-card__reviews-col"
  />
</div>
`;
